packages/apps/dashboard/client/package.json

```diff
@@ -10,7 +10,8 @@
     "test": "vitest -u",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "vercel-build": "yarn workspace @human-protocol/sdk build && yarn build"
```

Comment:
Why it was working beforev?
IMU [we run sdk build as a postinstall script](https://github.com/humanprotocol/human-protocol/blob/34d5e5ad3a9b092b23d8d2a632bf25f46ae0a553/package.json#L12)

@flopez7

Reply:
Answer on the ☝🏻 : we hadn't been using sdk in Dashboard client before